docs/hydra/5min-tutorial.md

```diff
@@ -6,10 +6,10 @@ title: 5 Minute Tutorial
 To get started quickly, we provide a Docker Compose based example for setting up ORY Hydra, a PostgreSQL instance
 and an exemplary User Login & Consent App. You need to have the latest Docker as well as Docker Compose version installed.
 
-<img src="../../images/docs/hydra/oauth2-flow.gif" alt="OAuth2 Flow">
+<img src="../../website/static/images/docs/hydra/oauth2-flow.gif" alt="OAuth2 Flow">
```

Comment:
This is also wrong - the right URL is: `img src="/website/static/images/docs/hydra/oauth2-flow.gif"`

Reply:
Thanks for pointing out. Done!